CanvasColorSpaceProposal.md

```diff
@@ -218,6 +220,8 @@ Authors of games and imaging apps are expected to be enthusiastic adopters.
 
 * Should context creation throw on an unrecognized, non-undefined creation attribute?
 
+* The [Media Query APIs](https://www.w3.org/TR/mediaqueries-4/) use the names "p3" and "rec2020" instead of "display-p3" and "rec-2020". This divergence could be confusing.
```

Comment:
Would this be a good opportunity to align them? If this pull request added "p3" instead of "display-p3" it would reduce divergence.

Reply:
Both "p3" and "display-p3" have left the barn (WebKit ships it for CSS colors).

I'd prefer to align with CSS color spec. If we change that, then I'm open to changing this.

Reply:
Could you add a reference to the CSS color spec then to make it clearer that this aligns with that spec, and the Media Query APIs are the odd one out?

Reply:
Updated with a link to the CSS Color Level 4.